release.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+set -e
+
+if [ $# -eq 0 ]
+then
+	echo "No version supplied. Run \`make release VERSION=X.X\`"
+	exit 1
+fi
+
+version=$1
+
+if ! echo $version | grep -Eq "^[0-9]+\.[0-9]+\.[0-9]+$"
+then
+	echo "Invalid version: ${version}"
+	echo "Please specify a semantic version with no prefix (e.g. X.X.X)."
+	exit 1
+fi
+
+echo "Listing supported resources to ${version}-resources.txt"
+go build -o cmd ./tfplan2cai/cmd
+./cmd list-supported-resources > ${version}-resources.txt
```

Comment:
Should we name the file in this way? Since the version is different from tfv in that driver folder.

Reply:
Adding a `tgc` prefix like you did manually seems reasonable to me 👍

Reply:
Done